README.md

```diff
@@ -8,3 +8,19 @@ And some more text.
 fjghdsfkjg
 
 Here's some more text.
+
+Now I've been able to do a little more on GitHub.
+
+Today is President's Day.
+
+I'm working with JQuery too.
+
+Its freezing outside.
+
+There may be 6 - 8 inches.
+
+2. Today is now Tuesday.  There is ice on the sidewalk. I'm going to try to add, commit, and push again. 
```

Comment:
You forgot to mention in this line that you need to go out and buy a snow shovel so you can clear the sidewalk.